aips/aip-8.md

```diff
@@ -0,0 +1,57 @@
+---
+aip: 9
+title: Token Standard Reserved Properties
+author: areshand
+discussions-to: https://github.com/aptos-foundation/AIPs/issues/28
+Status: Draft
+last-call-end-date (*optional):
+type: Standard (framework)
+created: 01/06/2023
+---
+
+## Summary
+
+This proposal introduces framework-reserved properties to
+
+1. prevent properties used by the token framework being changed by creators freely.
+2. reserving properties that can be extended to support programmable token behavior ex: token freezing, soulbound token, etc
+
+## Motivation
+
+We have existing token properties used by our token standard to control who can burn the token. However, when the token’s default properties are mutable, creators can add these control properties after the token has been minted. The creator can burn these tokens from collectors. This is a known issue called out in token standard to set the token default properties to be immutable as a best practice.  To prevent this entirely, this proposal is to make it infeasible to update the control properties after the token creation.
```

Comment:
```suggestion
We have existing token properties used by our token standard to control who can burn the token. However, when the token’s default properties are mutable, creators can add these control properties after the token has been minted. The creator can burn these tokens from collectors. This is a known issue called out in the token standard to set the token default properties to be immutable as a best practice.  To prevent this entirely, this proposal is to make it infeasible to update the control properties after the token creation.
```
please add link to the token standard